platform/ios/scripts/publish.sh

```diff
@@ -35,4 +35,15 @@ zip -r ../${ZIP} *
 step "Uploading ${ZIP} to s3…"
 REPO_NAME=$(basename $TRAVIS_REPO_SLUG)
 aws s3 cp ../${ZIP} s3://mapbox/$REPO_NAME/ios/builds/ --acl public-read
-echo http://mapbox.s3.amazonaws.com/$REPO_NAME/ios/builds/${ZIP}
+echo "URL: https://mapbox.s3.amazonaws.com/$REPO_NAME/ios/builds/${ZIP}"
+
+#
+# update nightly
+#
+if [[ ${PUBLISH_VERSION} =~ "nightly" ]]; then
+    step "Updating ${PUBLISH_VERSION} to ${PUBLISH_STYLE}…"
+    GENERIC_NIGHTLY_FILENAME="mapbox-ios-sdk-${PUBLISH_VERSION}.zip"
+    aws s3 cp \
+        s3://mapbox/$REPO_NAME/ios/builds/${ZIP} \
+        s3://mapbox/$REPO_NAME/ios/builds/${GENERIC_NIGHTLY_FILENAME} --acl public-read
```

Comment:
I couldn’t make [redirects](http://docs.aws.amazon.com/AmazonS3/latest/dev/how-to-page-redirect.html) work — we’re not strictly a “website” and aws is simply passing the `x-amz-website-redirect-location` header in GET requests, returning HTTP 200 and a blank file instead of HTTP 301 and the redirected file.

```
$ aws s3api put-object --bucket mapbox --key mapbox-gl-native/ios/builds/mapbox-ios-sdk-nightly-dynamic.zip --acl public-read --website-redirect-location /mapbox-ios-sdk-nightly-dynamic-2017-03-09.zip
```

```
$ curl -I -L https://mapbox.s3.amazonaws.com/mapbox-gl-native/ios/builds/mapbox-ios-sdk-nightly-dynamic.zip 
HTTP/1.1 200 OK
x-amz-id-2: Ub2UZ8qnAymsFa8eQUxPQZakb9Pwfb1TspVyl1kvulVT32uN6oZZMv2pO13eKiA9B5yPoxQMSig=
x-amz-request-id: A66C36AF51BB9C8E
Date: Fri, 10 Mar 2017 20:50:27 GMT
Last-Modified: Fri, 10 Mar 2017 20:50:23 GMT
ETag: "d41d8cd98f00b204e9800998ecf8427e"
x-amz-website-redirect-location: /mapbox-ios-sdk-nightly-dynamic-2017-03-09.zip
Accept-Ranges: bytes
Content-Type: application/zip
Content-Length: 0
Server: AmazonS3
```